### PR TITLE
FIX - Pip 10 support for importing PipSession and parse_requirements

### PR DESCRIPTION
**Pull request by roshanguptamca**
OPT - upgrade python 3.6

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -2,29 +2,84 @@
 addopts=--tb=short
 
 [tox]
-envlist = py27,py35,isort-check,isort-fix,lint,docs
+envlist =
+    py27-{pip9,pip10}
+    py35-{pip9,pip10}
+    py36{pip9,pip10}
```
Comment (jeduden) [inline tox.ini]: could you also add a dash here for consistency ?

File: tox.ini [hunk 2]
```diff
@@ -2,29 +2,84 @@
...
+basepython = python3.6
+commands =
+   {[general]commands}
+
+[testenv:py36{pip9,pip10}]
+basepython = python3.6
+commands =
+    {[general]commands}
+
+[testenv:py36]
```
Comment (jeduden) [inline tox.ini]: this is not needed right ?

File: tox.ini [hunk 3]
```diff
@@ -2,29 +2,84 @@
...
+    lint
+    docs
+
 skipsdist = true
 
+basepython =
+    py27-{pip9,pip10}: python2.7
+    py35-{pip9,pip10}: python3.5
+    py36{pip9,pip10}: python3.6
+    py36: python3.6
```
Comment (jeduden) [inline tox.ini]: remove this line

File: tox.ini [hunk 4]
```diff
@@ -2,29 +2,84 @@
...
 
+[general]
+commands =
+     py.test tests
+deps =
+    -rrequirements/requirements-base.txt
+    -rrequirements/requirements-testing.txt
+
+[testenv:py27-{pip9,pip10}]
+basepython = python3.6
```
Comment (jeduden) [inline tox.ini]: this needs to be python2.7

File: tox.ini [hunk 5]
```diff
@@ -2,29 +2,84 @@
...
+commands =
+    {[general]commands}
+
+[testenv:py35-{pip9,pip10}]
+basepython = python3.6
```
Comment (jeduden) [inline tox.ini]: and this python3.5

File: tox.ini (hunk 4 above)
Comment (roshanguptamca) [reply tox.ini]: Done

File: tox.ini (hunk 5 above)
Comment (roshanguptamca) [reply tox.ini]: done

**Pull request merged**